docs/get-started/alerting.md

```diff
@@ -129,48 +227,65 @@ templates:
 ## Test alert expressions
 If you want to create custom templates, you can test the MetricsQL expressions for your custom template in the **Explore** section of PMM. Here you can also query any PMM internal database.
 
-
 To test expressions for custom templates:
 
 1. On the side menu in PMM, choose **Explore > Metrics**.
 2. Enter your expression in the **Metrics** field and click **Run query**.
```

Comment:
we need to tell them to make sure to verify that the field on the very right which can be either Builder or Code" They have to confirm it is CODE or change it it CODE so they can paste the expression in there - if it is BUILDER there is no field to paste the expression into (this should be step #2)